dash/response.py

```diff
@@ -0,0 +1,160 @@
+"""
+Extended Flask Response for use inside callbacks
+"""
+import collections
+import json
+import plotly
+from flask import Response
+from . import exceptions
+from .development.base_component import Component
+
+
+# pylint: disable=too-many-ancestors
+class DashResponse(Response):
+    """
+    Flask Response extended with option to convert a regular response to
+    valid Dash json-encoded component.
+
+    Return a `DashResponse` object from a Dash callback in order to set
+    other properties of the response like headers or cookies.
+    """
+    def __init__(self, response, **kwargs):
```

Comment:
Should be `output_value` instead of `response` for uniformity between the arg and the instance attribute.